content/en/core/overview/translations.md

```diff
@@ -8,49 +8,45 @@ relatedContent: >
   apps/reference/translations
 ---
 
-Apps built with CHT Core are localized so that users can use it in the language of their choice. It is currently available in English, French, Hindi, Nepali, Spanish, and Swahili. The goal of this doc is to help our team manage these and future translations.
+Apps built with CHT Core are localized so that users can use it in the language of their choice. Languages supported out of the box are English, French, Nepali, Spanish, and Swahili. The goal of this doc is to help our team manage these and future translations.
```

Comment:
Is "out of the box" too colloquial?

Reply:
`Out of the box` sounds like no effort is required to keep them up to date. Nevertheless, I can't think of another more suitable term. Maybe "by default", "standard"?

Reply:
I was indicating that no effort is required for implementations to use them. I like "by default" I think...